src/briefcase/integrations/android_sdk.py

```diff
@@ -126,6 +126,64 @@ def verify_android_sdk(command):
     return sdk
 
 
+def _get_choices_and_names(running_devices, all_emulators, avd_name_by_device_id):
+    '''Input:
+
+    - `running_devices`: Dict from ADB device ID to dict with 'model' property.
+    - `all_emulators`: List of emulator device names.
+    - `avd_name_by_device_id`: A dict from ADB device ID to AVD name.
```

Comment:
Could `all_emulators` and `avd_name_by_device_id` be merged into a single dict with a None value if we don't know the device ID? 

If we can't - can we rename `avd_name_by_device_id` to a more functional name - like "active_emulators"?